src/pages/home/report/comment/TextWithEmojiFragment.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import type {StyleProp, TextStyle} from 'react-native';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as DeviceCapabilities from '@libs/DeviceCapabilities';
+import {splitTextWithEmojis} from '@libs/EmojiUtils';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+
+type ComponentProps = {
```

Comment:
```suggestion
type TextWithEmojiFragmentProps = {
```

Also, please add descriptions for each prop